webknossos/examples/apply_merger_mode.py

```diff
@@ -0,0 +1,91 @@
+from pathlib import Path
+from shutil import copyfileobj
+from typing import Tuple, cast
+from urllib.request import urlopen
+from zipfile import ZipFile
+
+import fastremap
+
+import webknossos as wk
+
+
+def main() -> None:
+
+    ####################################
+    # Opening a merger mode annotation #
+    ####################################
+
+    nml = wk.Skeleton.load(Path("testdata/annotations/l4_sample__merger_mode.nml"))
+
+    ###############################################
+    # Download and open the corresponding dataset #
+    ###############################################
+
+    copyfileobj(
+        urlopen("https://static.webknossos.org/data/l4_segmentation.zip"),
+        open("testdata/l4_segmentation.zip", "wb"),
+    )
```

Comment:
I assume the zip file contains only partial cutout / small bounding box of the segmentation. Can one specify a bounding box with `wk.Dataset.download()`? Otherwise the whole L4 segmentation will probably be downloaded :-/ ?

Reply:
Yes, you can specify a bounding box (currently the arg is `bbox`, renaming to `bounding_box` would probably be appropriate): https://docs.webknossos.org/api/webknossos/dataset/dataset.html#Dataset.download